.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Unit/Integration tests
```

Comment:
This is just a file rename for better clarity. Also added env-var setup workflow.